scripts/local_testnet/README.md

```diff
@@ -1,201 +1,84 @@
 
+1. Install [Kurtosis](https://docs.kurtosis.com/install/). Verify that Kurtosis has been successfully installed by running `kurtosis version` which should display the version.
 
-MacOS users need to install GNU `sed` and GNU `grep`, and add them both to `PATH` as well.
-
-The first step is to install Rust and dependencies. Refer to the [Lighthouse Book](https://lighthouse-book.sigmaprime.io/installation-source.html#dependencies) for installation. We will also need [jq](https://jqlang.github.io/jq/), which can be installed with `sudo apt install jq`.
-
-Then, we clone the Lighthouse repository:
-```bash
-cd ~
-git clone https://github.com/sigp/lighthouse.git
-cd lighthouse
-```
-We are now ready to build Lighthouse. Run the command:
-
-```bash
-make
-make install-lcli
-```
-
-This will build `lighthouse` and `lcli`. For `geth` and `bootnode`, go to [geth website](https://geth.ethereum.org/downloads) and download the `Geth & Tools`. For example, to download and extract `Geth & Tools 1.13.1`:
-
-```bash
-cd ~
-curl -LO https://gethstore.blob.core.windows.net/builds/geth-alltools-linux-amd64-1.13.1-3f40e65c.tar.gz
-tar xvf geth-alltools-linux-amd64-1.13.1-3f40e65c.tar.gz
-```
-
-After extraction, copy `geth` and `bootnode` to the `PATH`. A typical directory is `/usr/local/bin`.
-
-```bash
-cd geth-alltools-linux-amd64-1.13.1-3f40e65c
-sudo cp geth bootnode /usr/local/bin
-```
-
-After that We can remove the downloaded files:
-
-```bash
-cd ~
-rm -r geth-alltools-linux-amd64-1.13.1-3f40e65c geth-alltools-linux-amd64-1.13.1-3f40e65c.tar.gz
-```
-
-We are now ready to start a local testnet.
+1. Install [yq](https://github.com/mikefarah/yq). If you are on Ubuntu, you can install `yq` by running `sudo apt install yq -y`.
 
 ## Starting the testnet
 
-To start a testnet using the predetermined settings:
+To start a testnet, from the Lighthouse root repository:
 
 ```bash
-cd ~
-cd ./lighthouse/scripts/local_testnet
-./start_local_testnet.sh genesis.json
+cd ./scripts/local_testnet
+./start_local_testnet.sh
 ```
 
-This will execute the script and if the testnet setup is successful, you will see "Started!" at the end. 
-
-The testnet starts with a post-merge genesis state. 
-The testnet starts a consensus layer and execution layer boot node along with `BN_COUNT`
-(the number of beacon nodes) each connected to a geth execution client and `VC_COUNT` (the number of validator clients). By default, `BN_COUNT=4`, `VC_COUNT=4`. 
+It will build a Lighthouse docker image from the root of the directory and will take an approximately 12 minutes to complete. Once built, the testing will be started automatically. You will see a list of services running and "Started!" at the end. 
+You can also select your own Lighthouse docker image to use by specifying it in `network_params.yml` under the `cl_image` key.
+Full configuration reference for kurtosis is specified [here](https://github.com/kurtosis-tech/ethereum-package?tab=readme-ov-file#configuration).
```

Comment:
The `ethereum-package` has been moved to ethpandaops

```suggestion
Full configuration reference for kurtosis is specified [here](https://github.com/ethpandaops/ethereum-package?tab=readme-ov-file#configuration).
```

Reply:
thanks!